### ISSUE TITLE
Quantizing the recoding trigger?

### ISSUE THREAD

**Issue by freddieventura** [adriananders/Quix-Audio-Recorder-M4L#1]
Hi,

First of all many many thanks for this device, is amazing, and just thanking you for this brick you have added to this massive construction set called Ableton.

Just an issue, is there a way to quantize the recording signal, meaning when I press record it gets triggered on the next Beat/bar , according to Live's quantize global values?
Am I missing an option on your device or you just wanted to keep it like old school gear?

Thanks

**Comment by adriananders** [comment]
Yes. Under "Recording Duration", there's an option called "Position". Select the starting point and how long you want the recording to go for. You can set it for the next bar by setting to that position. It doesn't currently have a relative start delay by beats/bars, only an absolute value. Give it a shot to see if this meets your needs. If not, we can talk about a future feature enhancement. Thanks for reaching out!
